tools/cellprofiler/image_math.xml

```diff
@@ -0,0 +1,301 @@
+                    <option value="Image">Image</option>
+                    <option value="Measurement">Measurement</option>
+                </param>
+                <when value="Image">
+                    <param name="select_the_@ORDER@_image" type="text" label="Enter the name of the @ORDER@ image">
+                        <expand macro="text_validator"/>
+                    </param>
+                </when>
+                <when value="Measurement">
+                    <expand macro="select_category" order="@ORDER@"/>
+                </when>
+            </conditional>
+        </xml>
+
+
+        <xml name="section_image_with_multiplier" token_title="First Image" token_order="first">
+            <section name="@ORDER@_image" title="@TITLE@" expanded="true">
+                <expand macro="image_without_multiplier" order="@ORDER@"/>
+                <param name="multiply_the_@ORDER@_image_by" type="float" value="1.0" label="Enter the number that you would like to multiply the @ORDER@ image by"/>
+            </section>
+        </xml>
+
+
+        <xml name="section_image_without_multiplier" token_title="First Image" token_order="first">
+            <section name="@ORDER@_image" title="@TITLE@" expanded="true">
+                <expand macro="image_without_multiplier" order="@ORDER@"/>
+            </section>
+        </xml>
+
+
+        <xml name="section_ops_result">
+            <section name="op_results" title="OPERATIONS ON THE RESULTS" expanded="false">
+                <param name="raise_the_power_of_the_result_by" type="float" value="1.0" label="Enter an exponent to raise the result to after the chosen operation"/>
+                <param name="multiply_the_result_by" type="float" value="1.0" label="Enter the number that you would like to multiply the above image by"/>
+                <param name="add_to_result" type="float" value="0.0" label="Enter a number to add to the result after the chosen operation"/>
+                <param name="set_values_less_than_0_equal_to_0" type="select" display="radio" label="Set the values less than 0 equal to 0?" help="Values outside the range 0 to 1 might not be handled well by other CellProfiler tools.">
+                    <option value="Yes">Yes</option>
+                    <option value="No">No</option>
+                </param>
+                <param name="set_values_greater_than_1_equal_to_1" type="select" display="radio" label="Set the values greater than 1 equal to 1?" help="Values outside the range 0 to 1 might not be handled well by other CellProfiler tools.">
+                    <option value="Yes">Yes</option>
+                    <option value="No">No</option>
+                </param>
+            </section>
+        </xml>
+    </macros>
+    <requirements>
+    </requirements>
+    <command detect_errors="exit_code">
+        <![CDATA[
+            python '$__tool_directory__/image_math.py' -p '$input_pipeline' -i '$inputs'
```

Comment:
input could be named, like input_configfile or something like that ... maybe people expect here a image or so